docs/reference/migration/migrate_8_0/network.asciidoc

```diff
@@ -0,0 +1,10 @@
+[float]
+[[breaking_80_network_changes]]
+=== Network changes
+
+[float]
+==== Removal of old network settings
+
+The `network.tcp.connect_timeout` setting was deprecated in 7.x and has been removed in 8.0. This setting
+was a fallback setting for `transport.connect_timeout`. To change the default connection timeout for client
+connections `transport.connect_timeout` should be modified.
```

Comment:
Ditto re deprecation comment.  Couldn't find this setting defined either, expected it to be in https://www.elastic.co/guide/en/elasticsearch/reference/7.x/modules-network.html
If it's elsewhere we should mark it as deprecated in the 7.x branch

Reply:
The network connect timeout setting appears to not be documented.